v2/component/launcher.go

```diff
@@ -697,3 +709,21 @@ func getExecutorOutput() (*pipeline_spec.ExecutorOutput, error) {
 
 	return executorOutput, nil
 }
+
+func (l *Launcher) openBucket() (*blob.Bucket, error) {
+	if l.bucketConfig.scheme == "minio://" {
+		sess, err := session.NewSession(&aws.Config{
+			Credentials:      credentials.NewStaticCredentials("minio", "minio123", ""),
+			Region:           aws.String("minio"),
+			Endpoint:         aws.String("minio-service.kubeflow:9000"),
```

Comment:
this is tricky, namespace can be different in different envs:

For KFP standalone & full Kubeflow, the default is `kubeflow` namespace.
However, for AI Platform Pipelines, the default is `default` namespace and it's very easy for users to choose any namespace.

So we have a few options:
1. use endpoint `minio-service:9000`, this assumes minio service is in the same namespace, so it works for KFP standalone and AI Platform Pipelines, but not for full Kubeflow.
2. use endpoint `minio-service.kubeflow:9000`, this works for KFP standalone and full Kubeflow, but not AI Platform Pipelines

If we choose one of the two, 1. is better now, because next full Kubeflow release is at least 3 months from now, so we have some time to prepare for it.

Ideally, this should be configurable using a configmap + a secret. We can add this consideration for full artifact repository support after v2 compatible mode MVP.

Reply:
done